lib/test_process.dart

```diff
@@ -112,7 +110,7 @@ class TestProcess {
   /// This is protected, which means it should only be called by subclasses.
   @protected
   TestProcess(Process process, this.description,
-      {Encoding encoding, bool forwardStdio = false})
+      {required Encoding encoding, bool forwardStdio = false})
```

Comment:
Are you sure this is required here? Or should it just default to `utf8` like above?

Reply:
Indeed it would be more practical! It is now defaulting to utf8 as you proposed.